manifests/k/KDE/Dolphin/Nightly/master-b6ec4b1c9/KDE.Dolphin.Nightly.installer.yaml

```
@@ -2,15 +2,15 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: KDE.Dolphin.Nightly
-PackageVersion: master-b6ec4b1c9
+PackageVersion: master-271
 Scope: machine
 InstallModes:
 - interactive
 - silent
 Installers:
 - Architecture: x64
   InstallerType: nullsoft
-  InstallerUrl: https://binary-factory.kde.org/job/Dolphin_Nightly_win64/271/artifact/dolphin-master-271-windows-msvc2019_64-cl.exe
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Dolphin_Nightly_win64/lastSuccessfulBuild/artifact/dolphin-master-271-windows-msvc2019_64-cl.exe

```

```suggestion
  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Dolphin_Nightly_win64/271/artifact/dolphin-master-271-windows-msvc2019_64-cl.exe
```
Why was this changed? `lastSuccessfulBuild` in the URL is only valid for 1 day (24 hours), compared to having the build number in the URL which would last for about 4 or 5 days, before needing to be replaced.

If someone doesn't regularly update this like KDE Connect, people will be unable to download due to the expired URL. There will also likely be issue spam from @wingetbot when the URLs are unavailable if it's not updated in a timely matter.

---

> http://github.com/microsoft/winget-pkgs/pull/33717/files/40276aa68283b421fcf494cc32d1360d524e05f5#r742082211.

Do observe "http://github.com/microsoft/winget-pkgs/pull/33718#discussion_r742010117" for explanation of why I am believing that this modification is advantageous.

---

Please do observe http://github.com/microsoft/winget-pkgs/pull/33717#issuecomment-959718502.